src/ProjectTemplates/Web.ProjectTemplates/content/Components-CSharp/.template.config/dotnetcli.host.json

```
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json.schemastore.org/dotnetcli.host",
+  "symbolInfo": {
+    "skipRestore": {
+      "longName": "no-restore",
+      "shortName": ""
+    },
+    "PWA": {
+      "longName": "pwa",
+      "isHidden": true
+    },
+    "UseServer": {
+      "longName": "use-server",
+      "isHidden": true
+    },
+    "UseClient": {
+      "longName": "use-client",
+      "isHidden": true
```

Everywhere else I think we used WebAssembly instead of client. 

```suggestion
    "UseWebAssembly": {
      "longName": "use-wasm",
      "isHidden": true
```